### PR TITLE
[CARBONDATA -525] Timestamp based testcases were fixed for CI's

### PR DESCRIPTION
**Pull request by deepakmehra10**

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/122/

**Comment by deepakmehra10** [comment]
@ravipesala Please review the PR on priority . This PR includes fixed test cases which were failing on Continuous Integration.

**Comment by chenliang613** [comment]
LGTM

File: core/src/test/java/org/apache/carbondata/scan/complextypes/PrimitiveQueryTypeTest.java [hunk 1]
```diff
@@ -37,12 +40,14 @@
 import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
+import static sun.security.krb5.internal.ktab.KeyTabConstants.keySize;
```
Comment (kumarvishal09) [inline core/src/test/java/org/apache/carbondata/scan/complextypes/PrimitiveQueryTypeTest.java]: Why this import is required?? You can create a constants in your test case

Comment (kumarvishal09) [inline core/src/test/java/org/apache/carbondata/scan/complextypes/PrimitiveQueryTypeTest.java]: Why this import is required?? You can create a constants in your test case

Comment (chenliang613) [inline core/src/test/java/org/apache/carbondata/scan/complextypes/PrimitiveQueryTypeTest.java]: Why you imported this constant?  you can select to define a constant.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/130/

**Comment by ravipesala** [comment]
LGTM

**Comment by deepakmehra10** [comment]
Hi @chenliang613 can you please review the changes made. My PR seems to be fine i.e all the commands are running fine but the build gets failed on CI . Can you please check.

**Pull request closed without merging**